src/game.cpp

```diff
@@ -2185,7 +2188,10 @@ void Game::playerUseItem(uint32_t playerId, const Position& pos, uint8_t stackPo
 	player->resetIdleTime();
 	player->setNextActionTask(nullptr);
 
-	g_actions->useItem(player, pos, index, item, isHotkey);
+	bool isItemUsed = g_actions->useItem(player, pos, index, item, isHotkey);
```

Comment:
is there any reason why you didnt include that code in useItem of actions.cpp?
We got a code for cooldowns there already:
https://github.com/otland/forgottenserver/blob/master/src/actions.cpp#L409
Moving it would save places changed